bookstore/clone.py

```diff
@@ -18,9 +18,8 @@
 class BookstoreCloneHandler(IPythonHandler):
     """Handle notebook clone from storage.
 
-    Provides API handling for ``GET`` and ``POST`` when cloning a notebook
-    from storage (S3). Launches a user interface cloning options page when
-    ``GET`` is sent.
+    Provides handling for ``GET`` requests when cloning a notebook
```

Comment:
Let's change line 19 to: Prepares a clone options page and populates the UI with clone option parameters.

Reply:
Sounds good — I'll have a docstring & error string changing commit en-route to also incorporate your improved phrasing of "Requires" around the errors below.